### PR TITLE
Fixes #31954 - Try to fix ApplicableContentHelper test

### PR DESCRIPTION
**Pull request by jturel**
I see this pretty often: https://ci.theforeman.org/blue/organizations/jenkins/katello-pr-test/detail/katello-pr-test/6212/tests

@ianballou i doubt this is the ultimate fix but I'm trying to change to code to fail earlier if something went wrong - like updating the db. it doesn't actually seem that beneficial. wondering if you have any thoughts on the failure above?

### CODE REVIEW

File: test/services/katello/applicability/applicable_content_helper_test.rb [hunk 1]
```diff
@@ -21,20 +22,16 @@ def bound_repos(host)
 
         def setup
```
Comment (jturel) [inline test/services/katello/applicability/applicable_content_helper_test.rb:23]: The setup method does a lot of create vs save and I wonder if that's intentional?

File: test/services/katello/applicability/applicable_content_helper_test.rb [hunk 2]
```diff
@@ -56,16 +53,6 @@ def setup
           Katello::RepositoryErratum.create(erratum_id: @erratum.id, repository_id: @repo.id)
         end
 
-        def teardown
```
Comment (jturel) [inline test/services/katello/applicability/applicable_content_helper_test.rb:59]: Locally this doesn't seem to make any tests fail..

File: test/services/katello/applicability/applicable_content_helper_test.rb (hunk 1 above)
Comment (ianballou) [reply test/services/katello/applicability/applicable_content_helper_test.rb:23]: Not intentional, perhaps it could be related.

**Review by ianballou — APPROVED** [review]
I say let's merge this as soon as we can (assuming the tests pass).  It would be good to see if that one test failure stops popping up.

**Comment by jturel** [comment]
Gonna run this a handful of times to see if that failure resurfaces

**Comment by jturel** [comment]
one pass!

[test katello]

**Comment by jturel** [comment]
two

[test katello]

**Comment by jturel** [comment]
3
[test katello]

**Comment by jturel** [comment]
4
[test katello]

**Comment by jeremylenz** [comment]
I think 4 is good enough! :rocket:

**Comment by jturel** [comment]
> I think 4 is good enough!

Gotta file a redmine now so it's gonna be 5 :laughing:

**Comment by jeremylenz** [comment]
Build finished. 768 tests run, 0 skipped, 0 failed.

[test katello]

**Comment by jturel** [comment]
[test katello]

**Comment by jturel** [comment]
Sorry I didn't see you kicked the tests for me :(

**Pull request merged**